micrometer-core/src/main/java/io/micrometer/core/instrument/logging/LoggingMeterRegistry.java

```diff
@@ -114,6 +115,7 @@ private Function<Meter, String> defaultMeterIdPrinter() {
     @Override
     protected void publish() {
         if (config.enabled()) {
+            loggingSink.accept("Step: " + config.step().getSeconds() + "s");
```

Comment:
I'll remove it for now.

I thought it would be interesting to log the step duration but it would actually only increase number of logs.\
Step duration can already be derived from delta_count & throughput of any metric.

Reply:
The step is logged on start for LoggingMeterRegistry (and any PushMeterRegistry). While the default behavior is that things are logged, users can customize the `loggingSink` to do whatever it wants with lines sent to it. So, sending a line with the step each time that's a different format than metrics lines is bound to cause issues for custom loggingSinks.